### ISSUE TITLE
Does the library still work?

### ISSUE THREAD

**Issue by PhilipNelson5** [arosequist/node-owlet#4]
I've tried using this library and a python one, https://github.com/craigjmidwinter/pyowlet, but I get the same error on both: `Invalid email or password`. Can @arosequist confirm that the library still works?

**Comment by angel12** [comment]
Nope, owlet changed their backend, breaking all 3rd party projects

**Comment by PhilipNelson5** [comment]
Thank you for the quick reply, that is very unfortunate to hear. I found [this old thread](https://community.home-assistant.io/t/owlet-smart-sock-baby-monitor/69998/31) and figured somethings had changed.
